### ISSUE TITLE
Sometimes "save" doesn't work 

### ISSUE THREAD

**Issue by danfloyd111** [jgraph/drawio#4638]
### Preflight Checklist

* [x ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/dev/CODE_OF_CONDUCT.md) that this project adheres to.
* [ x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

If you are technical, you should reporting bugs along the lines of https://marker.io/blog/how-to-write-bug-report. If you are not technical, we will make allowances, please try to make an effort to understand the process.

**Describe the bug**
Auto-save and manual save through the menu are inconsistent, sometimes they work and sometimes no. The shortcut "cmd+s" always fails without error messages.

**To Reproduce**
1. Do some modifications to an existing schema
2. Click on 'save' or wait for / trigger the auto-save
3. Wait for the message "All changes saved" to appear in the menu bar
4. Quit the application
5. Open the application again
6. The modifications are gone like nothing happened

**Expected behavior**
The modifications should be saved, if not possible for some reason an error message should be shown.

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 24.7.8

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.6.1
 - Browser Version [e.g. 22]

**Smartphone (please complete the following information):**

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
-

**Additional context**
I switched all the available "themes" but the issue persists.

**Comment by alderg** [comment]
Where did you save the file?

**Comment by danfloyd111** [comment]
> Where did you save the file?

In a local folder under my home directory

**Comment by alderg** [comment]
Is that directory synched with a cloud storage such as Google Drive, OneDrive or Dropbox?

**Comment by danfloyd111** [comment]
> Is that directory synched with a cloud storage such as Google Drive, OneDrive or Dropbox?

No, it's completely local, no iCloud either. Other than draw.io files, contains a '.git' file to sync it with a remote repository, but I'm not performing any git ops in between.

I created a new diagram outside that folder, still completely local and without binding this to any git repository: the issue is still here. The auto-save seems more consistent, it works 80% of the times. Saving with cmd+s is not working at all (no messages are shown) while saving through the menu triggers the "all changes are saved" messages but after quitting the application the modifications disappear.

**Comment by alderg** [comment]
Are you using the desktop version or the web app (in a browser)? Please include all the version information requested in the original issue template.

**Comment by danfloyd111** [comment]
> Are you using the desktop version or the web app (in a browser)? Please include all the version information requested in the original issue template.

Sorry, I thought I did that by leaving blank the "browser lines". Fixing it now.
I'm using the desktop version 24.7.8.
